eng/versioning/version_client.txt

```diff
@@ -383,7 +383,7 @@ com.azure.tools:azure-sdk-build-tool;1.0.0-beta.1;1.0.0-beta.2
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
-unreleased_com.azure:azure-ai-formrecognizer;4.0.0-beta.7
+unreleased_com.azure:azure-ai-formrecognizer;4.0.0
```

Comment:
should we remove this?

Reply:
once it released, I think we can remove it.